docs/validators/delegation-manager.md

```diff
@@ -96,6 +96,47 @@ SetMetadataTransaction {
 }
 ```
 
+An example for the `Data` field that sets the name to `"Elrond Staking"`, the website to `"elrond.staking"` and the keybase.io identifier to `"elrondstaking"` is: 
+```
+    "setMetaData" + 
+    "@456c726f6e64205374616b696e67"     //Elrond Staking
+    "@656c726f6e642e7374616b696e67"     //elrond.staking     
+    "@656c726f6e647374616b696e67"       //elrondstaking
+```
+
+:::important
+Setting the keybase.io identity of the staking pool in the metadata is the first step in connecting the delegation contract to a keybase.io identity. The second step is explained in the next chapter [display information](/validators/delegation-manager#display-information) where the inverse connection is made: from the keybase.io identity to the delegation contract address. 
+:::
+
+### Display Information
+
+To customize the information for your delegation contract that will be available in the lists displayed on the delegation pages both in Maiar and the webwallet some aditional information has to be added on the keybase.io account. Please fill in the **avatar picture** and edit the profile providing the **name** and **bio**. This information together with the **service fee, percentage filled** and **APR** will be displayed for every delegation contracton the delegation pages in the webwallet and Maiar. If these information cannot be found a generic logo and the delegation contract's contract address is displayed.
+
+In order to complete the matching between the delegation contract and keybase.io identity of the staking pool an empty file with the name set to the delegation contract's contract address has to be added in the `/public/<keybase.io identity>/elrond/` folder. An example for the path to the empty file for the `"elrondstaking"` keybase.io identity would be `public/elrondstaking/elrond/erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqr0llllsj732py`.
+
+:::important
+The second and final step in connecting the delegation contract and the keybase.io identity is finalized by adding the empty file that has the name equal to the delegation contract's address and certifying that the referenced keybase.io identity set in the [metadata](/validators/delegation-manager#metadata) also references the delegation contract address. This way the connection from both directions is sealed.
```

Comment:
The second and final step in connecting the delegation contract and the keybase.io identity is finalized by adding the empty file that has the name equal to the delegation contract's address and certifying that the referenced keybase.io identity set in the [metadata](/validators/delegation-manager#metadata) also references the delegation contract address.

add some commas:

The second and final step in connecting the delegation contract and the keybase.io identity is finalized by adding the empty file, which has the name equal to the delegation contract's address, and certifying that the referenced keybase.io identity, set in the [metadata](/validators/delegation-manager#metadata), also references the delegation contract address.